### DISCUSSION TITLE
自建服务器，用手机扫描二维码显示连接节点服务器失败？？

### DISCUSSION THREAD

**Discussion by pi-3p14** [chanify/chanify#86] [Q&A]
在树莓派上自建了chanify服务，在局域网和外网访问IP：port都有二维码显示，但用手机扫描后显示“连接节点服务器失败”，请教下这是为啥呢？？
运行chanify serve的时候有个提示
2021/08/22 21:18:31 Node server name: roubo, version: 1.2.1, serverless: false, node-id: AGD5ZXO

**Comment by wizjin** [comment]
你好，请问 `endpoint` 怎么是设置的。如果使用内网的话手机要能访问树莓派；外网的话 `endpoint` 要设置成外网 `ip`。

**Reply by pi-3p14** [reply]
![image](https://user-images.githubusercontent.com/47076945/130382256-4b0bc375-8abf-4bfa-b0c8-30dc243eac8e.png)
外网IP是可以访问的，通过frp转到树莓派上面
请帮忙看下这样写配置是否有问题呢？

**Comment by wizjin** [comment]
你好，我试了一下发现你贴的 ip 好像访问不到，你可以试一下在手机上的 safari 上是不是能访问
`http://182.254.149.54:7878/rest/v1/info` app 就是利用这个接口来添加服务器的。

**Reply by pi-3p14** [reply]
现在好像又可以了，我晕，第一次访问http://182.254.149.54:7878/rest/v1/info报了404错误
然后我重新试了一次，现在貌似可以了，也可以扫描二维码添加节点了，非常感谢哈，我再继续试试看

**Comment by wizjin** [comment]
不客气，有问题欢迎随时交流。
